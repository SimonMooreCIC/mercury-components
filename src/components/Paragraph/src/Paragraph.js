import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

import { GOTHAM } from '../../../constants';

const StyledParagraph = styled.p`
	margin: 0 0 20px;
	font-family: ${GOTHAM};
}`;

const Paragraph = props => <StyledParagraph {...props} />;

Paragraph.propTypes = {
  /** Text in the paragraph */
  children: PropTypes.node.isRequired,
};

export default Paragraph;