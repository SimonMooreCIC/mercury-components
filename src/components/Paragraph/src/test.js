import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';

import Paragraph from './';

describe(Paragraph, () => {
  const example = 'example';
  const wrapper = <Paragraph>{example}</Paragraph>;

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Paragraph>Example</Paragraph>, div);
  });

  it('matches wrapper snapshot', () => {
    expect(mount(wrapper)).toMatchSnapshot('wrapper mount');
  });
});
