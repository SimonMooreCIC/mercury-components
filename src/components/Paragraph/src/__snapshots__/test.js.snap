// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph matches wrapper snapshot: wrapper mount 1`] = `ReactWrapper {}`;
